### PR TITLE
Fix infinite reload

### PR DESCRIPTION
**Pull request by hassandraga**
* More details https://github.com/webui-dev/webui/issues/392

### CODE REVIEW

**Comment by hassandraga** [comment]
image

**Comment by hassandraga** [comment]
Converting this PR to draft because `webui.isConnected()` is added to WebUI 2.5.0. This PR can be merged when 2.5.0 get released.

**Comment by ttytm** [comment]
Thanks lots for the PR!

**Comment by hassandraga** [comment]
I removed the bind all workaround from `main.go` as it's not needed anymore after the bridge fix (https://github.com/webui-dev/webui/commit/93d78bf7c89c923bbbd47900092088cc167eba00).

File: main.go [hunk 1]
```diff
@@ -107,7 +107,10 @@ func main() {
 
 	w := ui.NewWindow()
 
-	ui.SetDefaultRootFolder("ui/build")
+	if !ui.SetDefaultRootFolder("ui/build") {
+		println("The folder `./ui/build` does not exist. Please run `bun run --cwd ui build`")
+		return
+	}
```
Comment (ttytm) [inline main.go]:
```suggestion
	if err := ui.SetDefaultRootFolder("ui/build"); err != nil {
		log.Fatalf("Ensure you have run `bun run --cwd ui build`. %v", err)
	}
```

I updated the go-webui dependecy to use the development version.
With the latest changes this wouldn't be a bool anymore.

The error value of set default root folder contains:
```go
		err = errors.New("error: failed to set the default root folder")
```

The PR might be ready to merge after this

**Pull request merged**